Dockerfile

```diff
@@ -1,16 +1,20 @@
+# Pull base image
 FROM node:boron
 
+# Set environment variables
 ENV INITSYSTEM on
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-        bc \
-        vim \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY package.json /tmp/package.json
-RUN cd /tmp && npm install
-RUN mkdir -p /usr/app && cp -a /tmp/node_modules /usr/app/
+# Update and install
+RUN apt-get update && apt-get install -y --no-install-recommends bc vim && \
+    rm -rf /var/lib/apt/lists/* && \
+    npm install --global resin-cli@6.6.11 && \
```

Comment:
We shouldn't install the CLI at all. The test runner should be modified to take a session token as an environment variable, which the user is responsible from obtaining.